### PR TITLE
Updating line-heights to be consistent and adding some simple reset

### PR DESCRIPTION
**Pull request by dbanksdesign**
#### Description of changes

Made line heights for controls (fields and buttons) consistent.

* Removed overrides in the docs messing up the default theme
* Updated the default theme line heights
* Added some simple CSS resets and global styles

#### Description of how you validated changes

Ran the docs site locally:

https://user-images.githubusercontent.com/321279/153547457-ab57099f-bc03-48d6-a4ea-fdc532f3c042.mp4

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are updated
- [x] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/ui/src/theme/css/styles.scss [hunk 1]
```diff
@@ -4,21 +4,57 @@
...
+
+@media (prefers-reduced-motion: reduce) {
+  html:focus-within {
+   scroll-behavior: auto;
+  }
+  
+  *,
+  *::before,
+  *::after {
+    animation-duration: 0.01ms !important;
```
Comment (0618) [inline packages/ui/src/theme/css/styles.scss:51]: Are we should we have to use `!important` here? If possible, we may prefer to save it as the last resource. https://css-tricks.com/when-using-important-is-the-right-choice/

This case feels like we need to use `!important`, but I'm not sure, so just to double-check 😁

**Review by joebuono — APPROVED** [review]
LGTM!

File: packages/ui/src/theme/css/styles.scss (hunk 1 above)
Comment (dbanksdesign) [reply packages/ui/src/theme/css/styles.scss:51]: Yea I should have added a comment here, we can maybe remove those styles. I got them from a CSS reset: https://piccalil.li/blog/a-modern-css-reset/ it does force our customers to make inclusive apps (respecting reduced motion settings), but maybe it is a little heavy-handed

File: packages/ui/src/theme/css/styles.scss [hunk 2]
```diff
@@ -4,21 +4,48 @@
...
 // needed due to very low specificity of 0 0 0
 * {
   box-sizing: border-box;
 }
 
+input,
+button,
+textarea,
+select {
+  font: inherit;
```
Comment (zchenwei) [inline packages/ui/src/theme/css/styles.scss:40]: Shouldn't this be inherited by default? Do we set anything that affect those elements?

**Review by zchenwei — APPROVED** [review]
LGTM!

**Pull request merged**